nemo/collections/nlp/models/language_modeling/megatron_gpt_sft_model.py

```diff
@@ -75,7 +82,7 @@
 __all__ = ["MegatronGPTSFTModel"]
 
 
-class MegatronGPTSFTModel(NLPAdapterModelMixin, MegatronGPTModel):
```

Comment:
This is breaking a lot of example scripts and tutorials using `MegatronGPTSFTModel.merge_cfg_with`